app/src/main/java/com/example/habitsmasher/PublicPrivateButtons.java

```diff
@@ -76,52 +66,51 @@ public PublicPrivateButtons(View view, boolean isPublic){
      * only one button may be clicked at a time.
      */
     public void setClickListeners(){
-        // on click listeners and logic behind the buttons switching
-        // TODO: Check interactions when the selected button is selected again
-        // preferably make it unselectable
-
+        // track whether the button is selected or not
         _publicButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 // if we check the public box, uncheck the private box
-                if (_privateSelected){
-                    _privateButton.performClick();
-                    _privateSelected = false;
+                if (_publicSelected){
+                    _publicSelected = false;
+                }
+                else {
                     _publicSelected = true;
-
                 }
-                // make only the private button clickable
-                _publicButton.setClickable(false);
-                _privateButton.setClickable(true);
             }
         });
 
+        // track whether the button is selected or not
         _privateButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                // if we check the private box, uncheck the public box
-                if (_publicSelected){
-                    _publicButton.performClick();
-                    _publicSelected = false;
+                if (_privateSelected){
+                    _privateSelected = false;
+                }
+                else{
                     _privateSelected = true;
-
                 }
-                // make only the public button clickable
-                _publicButton.setClickable(true);
-                _privateButton.setClickable(false);
             }
         });
     }
 
     /**
-     * Returns whether the public button is selected or not. Logically, if the public button is
-     * selected the private button is unselected, and vice versa.
-     * @return
+     * Returns whether the public button is selected or not.
+     * @return Status of the public button
      */
     public boolean isHabitPublic(){
         if (_publicSelected) return true;
         else return false;
     }
 
+    /**
+     * Returns whether the private button is selected or not
+     * @return Status of the private button
+     */
+    public boolean isHabitPrivate(){
+        if (_privateSelected) return true;
+        else return false;
+    }
+
```

Comment:
Figured if I had a method for public habits I should now have one for the private habits.